src/ChatWindow.css

```diff
@@ -0,0 +1,44 @@
+.chatWindow__container{
+    display: flex;
+    flex:5;
+    overflow: scroll;
+    flex-direction: column;
+    background-image: url("https://user-images.githubusercontent.com/15075759/28719144-86dc0f70-73b1-11e7-911d-60d70fcded21.png");
```

Comment:
when you use 
background-image also use background-repeat and background-position